twitter.css

```diff
@@ -0,0 +1,290 @@
+  height: 100vh;
+}
+
+.responsive-rows {
+  display: grid;
+  grid-auto-rows: repeat(auto-fit, minmax(800px, 1fr));
+}
+
+/* Layout */
+
+.resources-sidebar {
+  width: 275px;
+}
+
+.twitter-logo {
+  width: 55px;
+  height: 45px;
+  margin: 0;
+  padding: 10px;
+}
+
+.side-bar-menu {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  padding: 20px;
+  cursor: pointer;
+}
+
+.side-bar-menu h2 {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  padding: 20px;
+}
+
+.side-bar-menu :hover {
+  background-color: var(--twitter-background);
+  border-radius: 30px;
+  color: var(--twitter-color);
+  transition: color 100ms ease-out;
+}
+
+.side-bar-menu.active {
+  color: var(--twitter-color);
+}
+
+.side-bar-button__tweet {
+  width: 100%;
+  background-color: var(--twitter-color);
+  color: white;
+  border: none;
+  font-weight: 900;
+  border-radius: 30px;
+  margin-top: 20px;
+  height: 50px;
+}
+
+/* left side-bar */
+
+/* Feed Section */
+a {
+  text-decoration: none;
+
+}
+
+.scroll {
+  overflow: scroll;
+}
+
+.box-md {
+  height: 56px;
+}
+
+.main-box {
+  border: 2px solid var(--twitter-background);
+  margin: 3px;
+  min-height: 56px;
+  position: sticky;
+}
+
+.post-box {
+  border: 2px solid var(--twitter-background);
+  margin: 3px;
+  min-height: 300px;
+  overflow: scroll;
+}
+
+
+.menu-list {
+  list-style-type: none;
+  padding: 20px;
+  margin: 0;
+}
+
+.menu-list a {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  /* padding: 20px; */
+  color: black;
+}
+
+.menu-list img {
+   font-weight: 800;
+   font-size: 20px;
+   margin-right: 10px;
+   padding: 30px;
+   padding-bottom: 0px;
+  }
+
+.active {
+  color: var(--twitter-color)
+}
+
+.menu-list  :hover {
+  /* background-color: var(--twitter-background);
+  border-radius: 40px; */
+  color: var(--twitter-color);
+  transition: color 100ms ease-out;
+
+}
+
+.feed-menu-icon {
+  width: 25px;
+  height: auto;
+}
+
+menu-list-link {
+
+}
+
+/* FEED */
+.profile-feed-pic img {
+  height: 40px;
+  border-radius: 50%;
+  object-fit: contain;
+  
+}
+
+span img {
+  height: 40px;
+  border-radius: 30px;
+  object-fit: contain;
+}
+
+/* POST */
+.post__badge {
+  font-size: 14px !important;
+  color: var(--twitter-color);
+  margin-right: 5px;
+}
+
+.post__header-Special {
+  font-weight: 600;
+  font-size: 12px;
+  color: gray;
+}
+
+.post__headerDescription {
+  margin-bottom: 10px;
+  font-size: 15px;
+}
+
+.post__footer {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+}
+
+
+.post__body img {
+  width: 100%;
+  object-fit: contain;
+  border-radius: 20px;
+}
+
+.post {
+  border: 1px solid var(--twitter-background);
+  padding: 4px;
+}
+
+.post-tweet {
+  background-color: var(--twitter-color);
+  color: white;
+  border: none;
+  border-radius: 30px;
+  font-weight: 900;
+  width: 80px;
+  height: 40px;
+  margin-bottom: 40px;
+  margin-right: 20px;
+  cursor: pointer;
+  
+}
+
+.no-decoration {
```

Comment:
really like the use of a no-decoration class. could you have used this on the a tag above (line 103)

Reply:
Yes this was just something I saw someone do before so when i implemented it, it made everything easier.